tests/Unit/PrestaShopBundle/Service/DataProvider/Admin/CategoriesProviderTest.php

```diff
@@ -0,0 +1,474 @@
+<?php
+/**
+ * 2007-2019 PrestaShop SA and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace Tests\Unit\PrestaShopBundle\Service\DataProvider\Admin;
+
+use PHPUnit\Framework\TestCase;
+use PrestaShop\PrestaShop\Adapter\Module\Module;
+use PrestaShopBundle\Service\DataProvider\Admin\CategoriesProvider;
+use PrestaShop\PrestaShop\Core\Util\File\YamlParser;
+
+class CategoriesProviderTest extends TestCase
+{
+    /**
+     * @var CategoriesProvider
+     */
+    private $provider;
+
+    public function setUp()
+    {
+        $yamlParser = new YamlParser(_PS_CACHE_DIR_);
+        $prestashopAddonsConfig = $yamlParser->parse(_PS_ROOT_DIR_ . '/app/config/addons/categories.yml');
```

Comment:
You should copy this yml into the test resources, if we change this file for functional needs then the test will fail without valid reason